appveyor.yml

```diff
@@ -68,7 +68,9 @@ install:
   # for obvci_appveyor_python_build_env.cmd
   - cmd: conda install -c pelson/channel/development --yes --quiet obvious-ci
   # for msinttypes and newer stuff
-  - cmd: conda config --add channels conda-forge
+  # conda-forge may serve outdated versions of certain packages (e.g. conda
+  # itself), so append it to the end of the list.
+  - cmd: conda config --append channels conda-forge
```

Comment:
looks like this is not enough, I guess we also need a more recent version of conda itself (I guess what happens is that a recent conda-build exposes a bug in an old version of conda), see latest appveyor builds which fail with conda 4.2.13+conda-build 2.1.7 (vs my PR which uses 4.3.14/2.1.7)